resources/assets/components/Action/index.js

```diff
@@ -84,6 +85,23 @@ class Action extends React.Component {
             </li>
           </ul>
         </div>
+        {new Date(campaign.start_date.replace(/-/g, '/')) > Date.now() ? (
```

Comment:
Let's parse this safely like we did in #857, instead of relying on this regex replacement hack.

Reply:
oh great catch! I actually just deleted the regex since we don't even need that here!

Reply:
Hmm, how come we can use `new Date(campaign.start_date)` directly here but not elsewhere? Are you sure this works properly across browsers?

Reply:
Unrelated to this pull request, but noticed a typo that made it through [in #857](
https://github.com/DoSomething/rogue/pull/857/files#r267096683) when clicking through to compare with how we solved this there.

Reply:
> Hmm, how come we can use new Date(campaign.start_date) directly here but not elsewhere? Are you sure this works properly across browsers?

Shoot you're totally right, forgot to check in Safari. Updated!

Reply:
> Unrelated to this pull request, but noticed a typo that made it through in #857 when clicking through to compare with how we solved this there.

That is updated [here](https://github.com/DoSomething/rogue/pull/858/files#diff-0e7fbc8575f3965559b364378e6d6e54R144)!